src/libraries/System.CodeDom/src/System/CodeDom/Compiler/ExposedTabStringIndentedTextWriter.cs

```diff
@@ -47,14 +49,16 @@ internal string IndentationString
 
                     switch (_indent)
                     {
-                        case 0: _s = string.Empty; break;
                         case 1: _s = tabString; break;
                         case 2: _s = tabString + tabString; break;
                         case 3: _s = tabString + tabString + tabString; break;
                         case 4: _s = tabString + tabString + tabString + tabString; break;
                         default:
                             var args = new string[_indent];
-                            for (int i = 0; i < args.Length; i++) args[i] = tabString;
+                            for (int i = 0; i < args.Length; i++)
+                            {
+                                args[i] = tabString;
+                            }
```

Comment:
Presumably this could just be:
```C#
Array.Fill(args, tabString);
```

Reply:
fixed

Reply:
@hughbe be this doesn't seem to have fixed

Reply:
Turns out Array.Fill(...) is not supported on its target platforms (it is targeting netstandard2.0 and net461). @stephentoub are you OK with keeping the for loop as is or you have other suggestions?

Reply:
Oh yes thats why I reverted the change. sorry! Let me know what we should do

Reply:
Thanks, @hughbe, I don't think we need anything else